invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/Identifiers/PIDField/OptionalPIDField.js

```diff
@@ -207,7 +210,8 @@ class OptionalPIDFieldCmp extends Component {
 
         {canBeManaged && _isManagedSelected && (
           <ManagedIdentifierCmp
-            disabled={hasDoi && isEditingPublishedRecord}
+            // You cannot change the managed DOI option if you have already published with a local DOI
+            disabled={hasPublishedLocalDOI}
```

Comment:
We disable the unreserve button for locally managed DOIs only if there is a published version with one. In all the other cases, unreserve is there if the option is enabled.